test/puppetlabs/http/client/decompress_test.clj

```diff
@@ -31,12 +31,6 @@
       (DeflaterInputStream.)))
 
 (deftest gzip-compress-test
-  (testing "clojure gzip decompression"
-    (let [test-response {:headers {"content-encoding" "gzip"}
-                         :body    (gzip compressible-body)}
-          response (async/decompress test-response)]
-      (is (not (contains? (:headers response) "content-encoding")))
-      (is (= compressible-body (slurp (:body response))))))
```

Comment:
I think this test checks the ```decompress``` method (now removed) only rather than verifying that decompression correctly occurs somewhere during a call. That's tested here: https://github.com/puppetlabs/clj-http-client/pull/42/files#diff-8c3870f32d4f413c53fabcb18ad43b5aR520